xcffib.cabal

```diff
@@ -47,7 +47,7 @@ executable xcffibgen
   build-depends: base ==4.*,
                  xcffib,
                  xcb-types >= 0.14.0 && < 0.15,
-                 language-python >= 0.5.8 && < 0.6,
+                 language-python >= 0.5.8,
```

Comment:
I just added all these in da42eacad5d6cc17eb99b739d745deeaf9d9e951 because the release uploader complained, so I suspect we need to keep them.

Reply:
Agreed, can revert.

Reply:
I get these still:

`[21 of 23] Compiling Language.Python.Version3.Parser.Parser ( dist/build/Language/Python/Version3/Parser/Parser.hs, dist/build/Language/Python/Version3/Parser/Parser.o, dist/build/Language/Python/Version3/Parser/Parser.dyn_o )
dist/build/Language/Python/Version3/Parser/Parser.hs:7579:4: error: [GHC-83865]
    • Couldn't match type ‘Happy_GHC_Exts.Int32#’
                     with ‘Happy_GHC_Exts.Int#’
      Expected: Happy_Int
        Actual: Happy_GHC_Exts.Int32#
    • In the expression: Happy_GHC_Exts.indexInt32OffAddr# arr off
      In an equation for ‘happyIndexOffAddr’:
          happyIndexOffAddr (HappyA# arr) off
            = (Happy_GHC_Exts.indexInt32OffAddr# arr off)
     |
7579 |   (Happy_GHC_Exts.indexInt32OffAddr# arr off)
     |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Error: [Cabal-7125]
Failed to build language-python-0.5.8 (which is required by exe:xcffibgen from xcffib-1.5.0). See the build log above for details.
`

When I searched I found bugs you had commented on over on the haskell side. The fix I had was to pull a fixed pin from here [git@github.com:bramvdbogaerde/language-python.git ](https://github.com/bramvdbogaerde/language-python/tree/fix-happy) and set a cabal.project.local pointing at it - bear in mind I know next to nothing about haskell, so this was a punt which worked, and I was just waiting for that fix to go in.

Reply:
got it - thanks for the tip there.